README.md

```diff
@@ -130,3 +131,7 @@ like:
 ## Getting involved
 
 Please refer to our [contribution guidelines](CONTRIBUTING.md).
+
+## Contact
+
+You can mention our twitter account [@Validaria_](https://twitter.com/validaria_)
```

Comment:
Maybe we want to add social media badges on the top instead and enter our validaria email address here?

Reply:
This one for example:
```markdown
![Twitter Follow](https://img.shields.io/twitter/follow/Validaria_?style=social)
```

Reply:
I don't like the idea of a auto-follow button.
It only works for people that happen to be logged in, all others are redirected to the login page instead of the tweets of the account...
Why just link to the account.
Following is only one click away from there (if the user is logged in) anyway.